config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
My only concern with having this as a bonus rather than a core exercise, is it contains info introducing a couple of things, stubs and example tests, and that these may get bypassed if the exercise doesn't need completing, does that make sense? 

> once you click on "complete" for hello world, you would also have Leap available.

Would this be with Leap as the second core exercise, or as another `bonus` exercise.
Currently the core structure for the first two exercises is this
```
hello-world
hamming
  +-- leap
  +-- gigasecond
```

> a nice "bonus" exercise

Yes, I think we could have a couple more of those, maybe `gigasecond` and `bob`, I'll look into it, I can think of a couple of changes to the config.json already now the structure is starting to settle in my head.